pages/common/llvm-dis.md

```diff
@@ -0,0 +1,16 @@
+# llvm-dis
+
+> Converts LLVM bitcode files into human-readable LLVM Intermediate Representation (IR).
+> More information: <https://www.llvm.org/docs/CommandGuide/llvm-dis.html>.
+
+- Print a bitcode file as LLVM IR to stdout:
+
+`llvm-dis {{path/to/input.bc}} -o -`
+
+- Convert a bitcode file to an LLVM IR file with the same filename:
```

Comment:
Does it replace the file, or create a new file with a different file extension alongside it?

Reply:
Creates a new file with a different extension alongside

Reply:
Ah, I see. Could we reword this a bit to clarify that it doesn't overwrite the input file please?

Reply:
I think it's already pretty clear, since it's obvious that a bitcode file and an LLVM IR file have different extensions.

Reply:
Very well. I'm not experienced with this command, so I'll defer to your judgement here @marchersimon